src/osdatahub/FeaturesAPI/features_api.py

```diff
@@ -46,12 +47,16 @@ class FeaturesAPI:
     }
 
     def __init__(self, key: str, product_name: str, extent: Extent,
-                 spatial_filter_type: str = "intersects"):
+                 spatial_filter_type: str = "intersects", filter_join_type: str = "And"):
+        if filter_join_type not in {"And", "Or"}:
```

Comment:
And and Or could be lowercase to be more pythonic

Reply:
Think I agree here, though obviously you'll need to use the capitalised versions in the actual filter. Maybe the spatial filter could be a property with a getter & setter? Also, out of interest, is there a reason to construct a set to check the value in line 51, as opposed to a tuple or list?

Reply:
N/A due as latest commit removes the argument. Marking as resolved.